.jshintrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Contacts doesn't use jshint as far as I can tell. Does this file have any effect?

Reply:
I read the Phpstorm documentation on how to fix this, and it was proposed you either add .jshintrc file, or just _"esversion": 6_, on the package.json. Almost every line of code will show _xxx is available in ES6 (use 'esversion: 6')_ if i dont have this file.

Reply:
What was the problem?

Reply:
You could disable jshint in PHPStorm. I did so too because it was constantly annoying me.

![grafik](https://github.com/nextcloud/contacts/assets/1479486/71f9b4be-0be2-4e9e-90f6-04a8c4361923)